File: pinyougou-manager-web/src/main/resources/static/js/service/typeTemplateService.js
//服务层
app.service('typeTemplateService',function($http){

    this.findPageTypeTemplate=function (page,size,searchEntity) {
        return $http.post("../typeTemplate/findPageTypeTemplate?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../typeTemplate/findOne?id="+id)
    }

    this.del=function (ids) {
        return $http.post("../typeTemplate/delTypeTemplate",ids)
    }

    this.add=function (entity) {
        return $http.post("../typeTemplate/addTypeTemplate",entity)
    }

    this.update=function (entity) {
        return $http.post("../typeTemplate/updateTypeTemplate",entity)
    }

    //所有模板
    this.findAllTypeTemplate=function () {
        return $http.get("../typeTemplate/findAllTypeTemplate")
    }

});

File: interface/src/main/java/com/kgc/pinyougou/service/TypeTemplateService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.TypeTemplate;
import com.kgc.pinyougou.utils.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface TypeTemplateService {

	List<TypeTemplate> findAllTypeTemplate();

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findPageTypeTemplate(Integer pageNum, Integer pageSize, TypeTemplate typeTemplate);

	Integer insertTypeTemplate(TypeTemplate typeTemplate);

	TypeTemplate findOne(Long id);

	Integer updateTypeTemplate(TypeTemplate typeTemplate);

	Integer deleteTypeTemplate(Long [] ids);
}
